src/app/home-page/widget/starred-box/starred-box.component.html

```diff
@@ -1,192 +1,204 @@
-    </mat-card-header>
-    <div id="total-starred" fxLayout="row" fxLayoutGap="1rem" class="size-small pb-3">
-      <div [ngPlural]="totalStarredWorkflows" id="starred-workflows" fxLayout="row" fxLayoutAlign="start center">
-        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" />0 Workflows</ng-template>
-        <ng-template ngPluralCase="=1"> <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" />1 Workflow</ng-template>
-        <ng-template ngPluralCase="other">
-          <img src="../assets/svg/star-on.svg" class="mr-1 site-icons-small" />{{ totalStarredWorkflows }} Workflows</ng-template
-        >
-      </div>
-      <div [ngPlural]="totalStarredTools" id="starred-tools" fxLayout="-mediumrow" fxLayoutAlign="start center">
-        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" />0 Tools</ng-template>
-        <ng-template ngPluralCase="=1"> <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" />1 Tool</ng-template>
-        <ng-template ngPluralCase="other">
-          <img src="../assets/svg/star-on.svg" class="mr-1 site-icons-small" />{{ totalStarredTools }} Tools</ng-template
-        >
-      </div>
-      <div [ngPlural]="totalStarredServices" id="starred-services" fxLayout="row" fxLayoutAlign="start center">
-        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" />0 Services</ng-template>
-        <ng-template ngPluralCase="=1"> <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" />1 Service</ng-template>
-        <ng-template ngPluralCase="other">
-          <img src="../assets/svg/star-on.svg" class="mr-1 site-icons-small" />{{ totalStarredServices }} Services</ng-template
-        >
-      </div>
-      <div [ngPlural]="totalStarredOrganizations" id="starred-orgs" fxLayout="row" fxLayoutAlign="start center">
-        <ng-template ngPluralCase="=0"> <img src="../assets/svg/star-off.svg" alt="star" class="mr-1" />0 Organizations</ng-template>
-        <ng-template ngPluralCase="=1"> <img src="../assets/svg/star-on.svg" alt="star" class="mr-1" />1 Organization</ng-template>
-        <ng-template ngPluralCase="other">
-          <img src="../assets/svg/star-on.svg" class="mr-1 site-icons-small" />{{ totalStarredOrganizations }} Organizations</ng-template
-        >
+  <div fxLayout="column" fxLayoutAlign="start stretch" class="h-100">
+    <div id="header-bar" fxLayout="row wrap" fxLayoutAlign="space-between">
+      <mat-card-header fxLayout="row">
+        <mat-card-title>Starred</mat-card-title>
+      </mat-card-header>
+      <div fxLayout="row wrap" fxLayoutGap="2rem" class="size-small pb-3">
+        <div fxLayout fxLayoutAlign="center center">
+          <mat-icon *ngIf="totalStarredWorkflows" class="mat-icon-sm">star</mat-icon>
+          <mat-icon *ngIf="!totalStarredWorkflows" class="mat-icon-sm">star_border</mat-icon>
```

Comment:
I think this can be simplified to `<mat-icon class="mat-icon-sm">{{ totalStarredWorkflows ? 'star' : 'star_border' }}</mat-icon>`. Same for tools, services and organizations

Reply:
Ahh good idea!